docker/build/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:18.04
+RUN apt-get update && \
+    apt-get install -y apt-transport-https git gnupg software-properties-common wget && \
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
+    add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" && \
+    apt-get install -y clang-8 make && \
+    apt-get remove --purge -y software-properties-common wget && \
+    apt-get autoremove --purge -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    echo '#!/bin/bash \n$(which clang-8) "$@"' > /usr/bin/clang && \
+    chmod a+x /usr/bin/clang && \
+    echo '#!/bin/bash \n$(which llvm-link-8) "$@"' > /usr/bin/llvm-link && \
+    chmod a+x /usr/bin/llvm-link
+
+COPY ./docker/build/ .
```

Comment:
Note these copies are relative to the root of the directory, so I as I point out in the README you have to build these images in the context of the repo root too.